Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -150,7 +152,10 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
     }
 
     private func rekeyIfExpired() async {
+        os_log("Checking if rekey is necessary...", log: .networkProtectionKeyManagement)
```

Comment:
Glad to see these tbh, I added them myself locally when I was debugging this but discarded the changes before review.